lib/bm_plate_creation.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+TIMES = 5
+
+purpose = PlatePurpose.first
+
+Rails.logger.level = :warn
+
+Benchmark.bmbm do |x|
+  x.report('construct!') { TIMES.times { purpose.create! } }
+end
```

Comment:
Is this for testing performance? Can you add a little comment at the top explaining it? Thanks

Reply:
No problem.